### ISSUE TITLE
[1, 2]

### ISSUE THREAD

**Issue by polkovnikov-ph** [estools/escodegen#260]
get rewritten into
```javascript
[
    1,
    2
]
```
which is not what one would expect rewriting the code. That's a small, innocent literal array. And yet I see no way to make `escodegen` output such lists as `[1, 2]`. Could you add it as a default behaviour and an option, please?

**Comment by michaelficarra** [comment]
Yes, this can be done by determining the formatting based on the size/content of the generated source text for each element. Should be pretty easy. PRs welcome.

**Comment by deviousasti** [comment]
+1

**Comment by amahfouz** [comment]
+1
I need this fix. Has it not been merged yet?
I am using 1.8.1 and I am getting the line breaks.
